doc/popsikey.txt

```diff
@@ -79,8 +80,8 @@ popsikey#register({prefix}, {maps}, {opts})
                     \ col: 'cursor',
                     \ }
 <
-		*Error	do not override filter or callback
-                Doing so will break the menu's key-hanlder
+		WARNING: do not override filter or callback; doing so will
+		break the menu's key-handler.
```

Comment:
Also updated this as the single `*` for highlighted as an error as it's an incomplete reference; not sure if that was intentional, but it looked very odd. The `WARNING` also gets highlighted.

Reply:
It was indeed intentional (cf. `:help help-writing`; `<Tab>*Error<Tab>text` is specially highlighted. But this is fine, too.